defaults/main.yml

```diff
@@ -1,10 +1,14 @@
 ---
 mysql_user_home: /root
+mysql_user_name: root
```

Comment:
Can you add in a line of docs/comments here explaining the difference between `mysql_user_*` and `mysql_root_*` options; just because it's not obvious, especially since they all default to `/?root` :)